Gemfile

```diff
@@ -49,31 +49,33 @@ gem 'devise'
 #gem 'devise-guests', '~> 0.3'
 
 # Avalon-specific
```

Comment:
Resolve the `avalon_workflow`, `active_encode`, and `mediainfo` gem tags.

Reply:
avalon_workflow and active_encode have been resolved.